problems/4.arrays.js

```diff
@@ -12,16 +13,31 @@ function firstAndLastAverage() {
 * @param {number[]} arr - The input array
 * @returns {number[]} - The duplicated array
 */
-function duplicateArray() {
-
+const duplicateArray = (arr) => {
+ let newArray = [];
+ newArray.push(arr, arr);
+ return newArray;
```

Comment:
Run and test your code. if arr was [1, 2, 3] we'd end up with [[1, 2,3 ], [1, 2, 3]]. 
Try looking into concat on MDN.